libsearch/util.go

```diff
@@ -87,7 +87,13 @@ func WriteFileAtomic(pathname string, content []byte) error {
 // it to lower case and keeping only the alphanumeric characters.
 func NormalizeKeyword(keyword string) string {
 	lowerKeyword := strings.ToLower(keyword)
+	normalizedKeyword := make([]rune, 0, len(lowerKeyword))
 
-	re := regexp.MustCompile("[^a-z0-9]")
-	return re.ReplaceAllString(lowerKeyword, "")
+	for _, c := range lowerKeyword {
+		if unicode.IsDigit(c) || unicode.IsLetter(c) {
+			normalizedKeyword = append(normalizedKeyword, c)
```

Comment:
Actually it looks like `strings.ToLower` just calls `unicode.ToLower` on each character anyway.  See https://golang.org/src/strings/strings.go?s=11848:11911#L432.

So the only advantage to calling it separately is we can avoid a second loop over the keyword.  Up to you.